src/client/watchClientChanges.js

```diff
@@ -46,6 +46,7 @@ const watchClientChanges = clientConfig => {
   server.listen(port, 'localhost', () => {
     console.log(`Starting webpack-dev-server on ${webpackDevServerUrl}`);
   });
+  return server;
```

Comment:
@jsepia this is the `webpackDevServer` object you are returning, not node's `httpServer` object. I think to be consistent, both client and server should return node's `httpServer` object? I think this is what you want rather than the `webpackDevServer` based on the types you added. Let me know what you think.

Reply:
Makes sense, and yes, this was the original intent. I did wonder if it made sense to return `WebpackDevServer` but there doesn't seem to be anything useful in it:

```ts
declare class WebpackDevServer {
    constructor(...);
    static addDevServerEntrypoints(...): void;
    listen(...): http.Server;
    close(callback?: () => void): void;
}
```